doc/services/misc.rst

```diff
@@ -34,3 +34,16 @@ this API is limited to creating the simplistic tokens needed to
 authenticate with the Google Core IoT infrastructure.
 
 .. doxygengroup:: jwt
+
+Identifier APIs
+***************
+
+UUID
+====
+
+Universally Unique Identifiers (UUID), also known as Globally Unique
+IDentifiers (GUIDs) are an open, industry standard [RFC
+9562](https://tools.ietf.org/html/rfc9562) 128 bits long identifiers
+intended to guarantee uniqueness across space and time.
```

Comment:
You are wrapping very aggressively here (72 characters?) --- 100 is fine/better :)

Also, you can be fancy with the Sphinx `:rfc:` role. Reworded + reformatted below:

```suggestion
Universally Unique Identifiers (UUID), also known as Globally Unique IDentifiers (GUIDs), are an
open industry standard (:rfc:`9562`) for 128-bit long identifiers intended to guarantee uniqueness
across space and time.
```